### PR TITLE
chore: Change the default color of the font in the button

### PR DESCRIPTION
**Pull request by hanzhenfang**
Hello，i find a small problem when i was accessing Apisix-website ，The text of this button does not seem to be displayed clearly.
image

Changes：I just changed the default display color of <a>.
😊

image

### CODE REVIEW

**Comment by guoqqqi** [comment]
There is a CI error, you can fix it here https://github.com/zeke/semantic-pull-requests#example-scenario @hanzhenfang

File: website/src/css/customTheme.css [hunk 1]
```diff
@@ -710,6 +710,10 @@ a:hover {
   border-width: 1.75px;
   padding: 5px 12px;
   transition: background 0.5s;
+
```
Comment (guoqqqi) [inline website/src/css/customTheme.css]:
```suggestion
```
remove the extra blank

**Comment by guoqqqi** [comment]
You can add some images to the description section to make it easier for others to see.

File: website/src/css/customTheme.css (hunk 1 above)
Comment (hanzhenfang) [reply website/src/css/customTheme.css]: ok i will

**Comment by SkyeYoung** [comment]
@hanzhenfang Hi, Maybe you should use the preset PR template like other PR, just like [this one](https://github.com/apache/apisix-website/pull/884). :D

**Comment by SkyeYoung** [comment]
@hanzhenfang Hi, why you closed this PR, it's a good job!

**Comment by guoqqqi** [comment]
Hi, @hanzhenfang This is a good change,  why turn it off

**Comment by hanzhenfang** [comment]
> 

@SkyeYoung  oh，sorry，it's my first pr in github，im not familiar with it，so sorry about that.i will open the pr again。

**Comment by hanzhenfang** [comment]
@guoqqqi 
oh，sorry，it's my first pr in github，im not familiar with it，so sorry about that.i will open the pr again。

**Review by guoqqqi — APPROVED** [review]
LGTM

**Review by juzhiyuan — APPROVED** [review]

**Review by yzeng25 — APPROVED** [review]

**Pull request merged**